### PR TITLE
feat(flicking): Support previewPadding resize

### PR DESCRIPTION
**Pull request by netil**
## Issue
#13

## Details
Support change of previewPadding option after the initialization

```js
var some = new eg.Flicking("#mflick", {
	previewPadding: [10,10]
});

// when changes previewPadding option from its original value
some.options.previewPadding = [20, 30];
some.resize();
```

## Preferred reviewers
@happyhj @sculove

### CODE REVIEW

File: src/flicking.js [hunk 1]
```diff
@@ -1189,24 +1202,73 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
...
+
+			// when device orientaion changes
+			some.resize();
+
+			// when changes previewPadding option from its original value
+			some.options.previewPadding = [20, 30];
+			some.resize(true);
 		 */
-		resize: function () {
+		resize: function (checkPadding) {
```
Comment (happyhj) [inline src/flicking.js]: Adding a bool parameter to an existing function makes code hard to read without the documentation.
http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html


```
some.resize(true);
```
```
some.resize({ checkPadding: true });  // <-- easier to guess
```
How about using object parameter instead?

Comment (mixed) [inline src/flicking.js]: I agree with @happyhj .

Comment (netil) [inline src/flicking.js]: I will remove receiving param to check and resize only padding and option padding is different.

**Comment by sculove** [comment]
LGTM

**Comment by mixed** [comment]
LGTM

**Pull request closed without merging**